### PR TITLE
Update Spanish.properties

### PR DESCRIPTION
**Pull request by ArchDuque-Pancake**

### CODE REVIEW

File: android/assets/jsons/translations/Spanish.properties [hunk 1]
```diff
@@ -785,28 +781,25 @@ Treasury deficit = Déficit de tesorería
 
 # Victory
 
-Science victory = Victoria científica
-Cultural victory = Victoria cultural
-Conquest victory = Victoria de dominación
- # Requires translation!
-Diplomatic victory = 
+Science victory = Victoria C  ientífica
```
Comment (SomeTroglodyte) [inline android/assets/jsons/translations/Spanish.properties:784]: "C  i"

File: android/assets/jsons/translations/Spanish.properties [hunk 2]
```diff
@@ -42,21 +42,17 @@ Wonder is being built elsewhere = Esta maravilla se está construyendo en otro l
...
 Cannot be purchased = No se puede comprar
 See also = Véase también
 Requires at least one of the following: = Requiere por lo menos uno de los siguientes:
- # Requires translation!
-Requires all of the following: = 
- # Requires translation!
-Leads to [techName] = 
- # Requires translation!
-Leads to: = 
+Requires all of the following: = Requiere todos los siguientes
```
Comment (SomeTroglodyte) [inline android/assets/jsons/translations/Spanish.properties:53]: El ":" hace parte del texto. Aparece en la Civilopedia, an una línea por si mismo, y quizás va mejor sin el ":" - como quieres.

File: android/assets/jsons/translations/Spanish.properties [hunk 3]
```diff
@@ -785,28 +781,25 @@ Treasury deficit = Déficit de tesorería
...
 You have been defeated. Your civilization has been overwhelmed by its many foes. But your people do not despair, for they know that one day you shall return - and lead them forward to victory! = Has sido derrotado. Tu civilización ha sido abrumada por sus muchos enemigos. Pero su gente no se desespera, porque saben que algún día regresarán, ¡y los guiarán hacia la victoria!
- # Requires translation!
-You have triumphed over your foes through the art of diplomacy! Your cunning and wisdom have earned you great friends - and divided and sown confusion among your enemies! Forever will you be remembered as the leader who brought peace to this weary world! = 
+You have triumphed over your foes through the art of diplomacy! Your cunning and wisdom have earned you great friends - and divided and sown confusion among your enemies! Forever will you be remembered as the leader who brought peace to this weary world! = ¡Has triunfado sobre tus adversarios con el arte de la diplomacia! Tu astucia y sabiduria te han ganado muchos amigos - ¡y división y confusión entre tus enemigos! ¡Serás siempre recordado como el lider que trajo paz a este lugrube mundo!
```
Comment (SomeTroglodyte) [inline android/assets/jsons/translations/Spanish.properties:802]: 👍  pero - "líder"?

**Comment by SomeTroglodyte** [comment]
`Turns until the next\ndiplomacy victory vote: [$turnsTillNextDiplomaticVote] = ` @xlenstra - yours - I think the unusual `$` in the translation _template_ does work, but - do we want to create a precedent here? Sorry I didn't see this earlier, i _know_ I read the comment to it...

File: android/assets/jsons/translations/Spanish.properties [hunk 4]
```diff
@@ -818,22 +811,14 @@ Branches completed = Ramas completadas
...
-Voted for = 
- # Requires translation!
-Vote for [civilizationName] = 
- # Requires translation!
-Continue = 
- # Requires translation!
-Abstained = 
- # Requires translation!
-Vote for World Leader = 
+Turns until the next\ndiplomacy victory vote: [$turnsTillNextDiplomaticVote] = Turnos hasta la siguiente\n votación de victoria diplomatica: [$turnsTillNextDiplomaticVote]
```
Comment (SomeTroglodyte) [inline android/assets/jsons/translations/Spanish.properties:814]: elección? "diplomática"?

File: android/assets/jsons/translations/Spanish.properties [hunk 5]
```diff
@@ -818,22 +811,14 @@ Branches completed = Ramas completadas
...
+Choose a civ to vote for = Escoje una civ por la cual votar
+Choose who should become the world leader and win a diplomatic victory! = ¡Escoje a quien deeberia convertirse en lider mundial y ganar una victoria diplomatica!
```
Comment (SomeTroglodyte) [inline android/assets/jsons/translations/Spanish.properties:816]: "debería"?

File: android/assets/jsons/translations/Spanish.properties (hunk 4 above)
Comment (ArchDuque-Pancake) [reply android/assets/jsons/translations/Spanish.properties:814]: What?

**Comment by ArchDuque-Pancake** [comment]
I have to do a grammar test again o_o ...

And check for grammar errors afterwards...

File: android/assets/jsons/translations/Spanish.properties (hunk 4 above)
Comment (ArchDuque-Pancake) [reply android/assets/jsons/translations/Spanish.properties]: Aaahh ya.
I'll ask over and see what others think.

**Pull request merged**